### DISCUSSION TITLE
VueFlow tool won't let me click to other resources

### DISCUSSION THREAD

**Discussion by HeadStudios** [laravel/nova-issues#5942] [Q&A]
Laravel Version: 9
Nova Version: 4.25.1
PHP Version: 8.2
Database Driver & Version: MySQL
Operating System and Version: Macbook
Browser type and version: Chrome latest
Reproduction Repository: https://github.com/HeadStudios/novaissue-5456-nonova

### Description:
After installing the VueFlowDev tools - clicking on any other resource does not work - expected behavior is if I want to click away from the resource I should be able to.

### Detailed steps to reproduce the issue on a fresh Nova installation:
1. Install new tool and setup VueTools:
https://codeshare.io/YLDbeQ
https://codeshare.io/r93y17

2. Setup the menu in NovaServiceProvider - https://codeshare.io/N3ljvV

Try to move to another resource 
Video for context: https://rrdevours.monster/storage/support/issue_vueflowtool.mp4

**Comment by crynobone** [comment]
Not familiar with VueFlowDev tools. [v4.26.0](https://nova.laravel.com/releases/4.26.0) upgrade Vue from 3.2 to 3.3 which requires all tools to recompile to avoid this issue. 

Since you are running on Laravel Nova 4.25.1 it could be possible VueFlowDev tools already compiled with Vue 3.3 and not possible to be used with 3.2

**Reply by HeadStudios** [reply]
Thank you! I can confirm updating to Vue 3.3. and updating Nova to latest version has fixed both problems.
